README.md

```diff
@@ -318,6 +318,36 @@ await service.initializePayment({
         container: 'walletWidget',
     },
 });
+
+```
+Also, PayPal (also known as PayPal Commerce Platform) requires specific options to initialize the PayPal Smart Payment Button on checkout page that substitutes a standard submit button
+
+````
+<!-- This is where the PayPal button will be inserted -->
+<div id="container"></div>
+service.initializePayment({
+    methodId: 'paypalcommerce',
+    paypalcommerce: {
+        container: '#container',
+        submitForm: () => {
+            service.submitOrder(
+                {
+                    payment: { methodId: 'paypalcommerce', }
+                }
+            );
+        },
+        onValidate: (resolve, reject) => {
+            const isValid = service.validatePaymentForm();
+            if (isValid) {
+                return resolve();
+            }
+            return reject();
+        },
+        onRenderButton: () => {
+            service.hidePaymentSubmitButton();
```

Comment:
Can you also add comments for `onRenderButton` and `onValidate`, and `submitForm` methods https://github.com/bigcommerce/checkout-sdk-js/pull/1017#discussion_r529058647
like 
```js
        //  A callback that is called right before render of a Smart Payment Button. It gets called when a buyer is eligible for use of the particular PayPal method. This callback can be used to hide the standard submit button.
        onRenderButton: () => {
        ...
        // this callback is used to define the state of the payment form, validate if it is applicable for submit
        onValidate: (resolve, reject) => {
        ...
        // A callback for submitting payment form that gets called when buyer approved PayPal payment
        submitForm: () => {
        ...
```

Reply:
Provider-specific instructions should be surfaced in the `docs` folder (e.g: https://github.com/bigcommerce/checkout-sdk-js/blob/master/docs/interfaces/paypalcommercepaymentinitializeoptions.md) rather than the top-level readme markdown file.

Reply:
@davidchin  do you suggest to move the entire section to the options?